controllers/dbaas.redhat.com/crunchybridgeinventory_controller.go

```diff
@@ -18,26 +18,28 @@ package dbaasredhatcom
 
 import (
 	"context"
-	"net/url"
```

Comment:
Please make/keep includes consistent with (core <line break> 3rd party <line break> org/local/project ) grouping used in the other go files. (one step in the right direction would use `goimports` on save over `gofmt`, as the former is a superset)